ccm_web/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# References:
+# - https://docs.docker.com/develop/develop-images/multistage-build/
+# - https://dev.to/chrsgrrtt/dockerising-a-next-js-project-1ck5
+
+# Base stage
+FROM node:14-slim AS base
+WORKDIR /base/
+COPY package*.json ./
+RUN npm install
+COPY . .
+
+ARG PORT
+EXPOSE ${PORT}
+
+# Build stage (build client and compile server to JS)
+FROM base AS build
+WORKDIR /build/
+COPY --from=base /base ./
+RUN npm run preprod
+
+# Prod stage
+FROM node:14-slim AS prod
+ENV NODE_ENV=production
+WORKDIR /app
+
+COPY --from=base /base/package.json ./
+RUN npm install --production
+
+COPY --from=build /build/dist/ ./
```

Comment:
the `dist` folder in the container needs to map to local disk space, since as we are keep changing text new file will be created but We need to delete it without going into container. We need to see the `dist` file otherwise we forget to delete it and container size increases and we run into space issues

Reply:
I DMed you to discuss this. This step will only be performed for production builds. Happy to explain. I could of course be doing something wrong, so I'd like some further explanation.